tests/ssl-opt.sh

```diff
@@ -695,6 +695,17 @@ run_test() {
     CLI_EXPECT="$3"
     shift 3
 
+    # Expected exit code for the server in response to SIGTERM
+    # Expect 1 for gnutls-serv, otherwise default to expecting 0
+    case $SRV_CMD in
+    gnutls-serv\ *)
+        SRV_EXPECTED_EXITCODE="1"
```

Comment:
While I normally favor long, explicit variable names, in this case, I think it should be called `SRV_EXPECTED_RET` (or `EXPECTED_SRV_RET`, for consistency with `SRV_RET`.